### DISCUSSION TITLE
Apple Cinema Display immediately too dark

### DISCUSSION THREAD

**Discussion by thedesignerdave** [MonitorControl/MonitorControl#1003] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

I installed monitor control on my Macbook pro to control the brightness of an external monitor. After instillation, I adjusted the brightness slider in the app. The second I touched the slider, my monitor when to about 40% brightness even though the slider bar shows that it's at full brightness. I've tried disconnecting, reseting v/pram on my mac. Removing and reinstalling monitor control and the monitor is stuck. I've even hooked it up to a different macboook pro that doesn't have Monitor control and it's still dark. 
Please save me from the darkside!

### Expected behavior

I expect the software to be able to adjust the brightness to 100%

### Environment Information (please complete the following information)

```markdown
- macOS version: BigSur 11.6.5
- Mac model:Macboook pro 15 inch 2017
- MonitorControl version:?
- Monitor(s): Apple cinema 27 thunderbolt display
- Monitor Cable(s)/Connection(s):Thunderbolt to usbc adapter
- Apple Silicon/M1 (yes or no):?
```

**Comment by waydabber** [comment]
Hi there,

try setting this to 255, that tends to help with older Apple displays which (in an undocumented way) support DDC commands for brightness control:

<img width="842" alt="Screen Shot 2022-04-12 at 17 24 02" src="https://user-images.githubusercontent.com/37590873/162997507-b1bbed4f-d8e0-49c4-8a69-0d16675ee49f.png">

**Reply by SkipLesion** [reply]
I had exactly this problem with my 14" M1 MacBook Pro and this solution worked brilliantly for my ancient but otherwise still excellent 2008 24" Apple LED Cinema Display - many thanks!

**Comment by thedesignerdave** [comment]
> Hi there,
> 
> try setting this to 255, that tends to help with older Apple displays which (in an undocumented way) support DDC commands for brightness control:
> 
> <img alt="Screen Shot 2022-04-12 at 17 24 02" width="842" src="https://user-images.githubusercontent.com/37590873/162997507-b1bbed4f-d8e0-49c4-8a69-0d16675ee49f.png">

Thanks for the suggestion but it didn't work.

**Comment by waydabber** [comment]
Well, connecting the display to a Mac that has an OS from before Apple just removed brightness control support supposedly fixes the issue. The problem is that these displays respond to DDC commands to some extent and depending on firmware and hardware revision they work or they just do this. Others reported that using Lunar by @alin23 did fix the problem (maybe because it uses different timings somewhat or repeats DDC commands more or differently - who knows). The problem is surely fixable and probably by tuning DDC commands things would work but I don't have a display like this to experiment.

for more info see these:

https://github.com/MonitorControl/MonitorControl/discussions/970
https://github.com/MonitorControl/MonitorControl/discussions/739

**Reply by thedesignerdave** [reply]
Thanks for the suggestions. I installed Luna and after resetting and resetting and resetting with no change... suddenly it worked. 
Looks lIke I'll be staying away from Monitor Control for my older monitor.

**Reply by waydabber** [reply]
Yes, that's the best you can do. If you disable hardware DDC control you can use software gamma control with the display to control brightness (but this will not change the backlight level). I contacted Lunar's developer about this but he says Lunar has a similar issue with some of the Cinema display. Sometimes control works after many tries but usually not.

**Comment by jimhuls** [comment]
I just learned of and installed MonitorControl at home and work. Works great at home on my 34" monitor but at work I have two 27" Apple Cinema Displays where they immediately went dark.

I entered in 255 for ddc max for brightness in the advanced settings and while no change occurred, I then unchecked "Brightness" and then checked it again....voila...full brightness is back.
